deploy_tests/cgap/cgap.py

```diff
@@ -7,60 +7,67 @@
 from deploy_tests.utils import build_url
 
 
-HOST = 'https://cgap.hms.harvard.edu/'
-
-
-# The following item types give no search result and thus 404, so we will not access their collection pages
-BAD_ITEM_TYPES = ['AnnotationField', 'EvidenceDisPheno', 'GeneAnnotationField', 'Image', 'QualityMetricPeddyqc',
-                  'QualityMetricWgsBamqc', 'TrackingItem', 'WorkflowMapping', 'WorkflowRun']
+# TODO: make arg
+HOST = 'https://cgap-msa.hms.harvard.edu'
```

Comment:
Do you have to edit this every time you want to try a new deploy? Why not make this a parameter?